### PR TITLE
added file info text for grid layout

### PR DESCRIPTION
**Pull request by firoze**
https://fieldwire.atlassian.net/browse/ME-361

in the original issue. the file info text was missing on grid view. this was originally by design of the filestack library (only icon/thumbnail was displayed in grid view). the textViews for file info have been updated to the grid layout, and padding adjusted for visibility. the text is set to single-line to avoid the ui being too cluttered. but this can be adjusted as needed

### CODE REVIEW

File: tester/src/main/java/com/filestack/android/demo/MainActivity.java [hunk 1]
```diff
@@ -69,7 +70,7 @@ public void launch(View view) {
         String mimeFilter = sharedPref.getString("mime_filter", null);
         List<String> mimeTypes = Arrays.asList(mimeFilter.split(","));
 
-        String apiKey = sharedPref.getString("api_key", null);
+        String apiKey = "ACGv8iMoCTXS27m716Xsqz";// sharedPref.getString("api_key", null);
```
Comment (davidnguyen2302) [inline tester/src/main/java/com/filestack/android/demo/MainActivity.java]: do we need the explicit key for this fix to work?

File: tester/src/main/java/com/filestack/android/demo/MainActivity.java [hunk 2]
```diff
@@ -81,8 +82,9 @@ public void launch(View view) {
 
         Theme theme = new Theme.Builder()
                 .title(sharedPref.getString("theme_title", null))
-                .accentColor(sharedPref.getInt("theme_accent", ContextCompat.getColor(this, R.color.colorAccent)))
-                .backgroundColor(sharedPref.getInt("theme_background", ContextCompat.getColor(this, R.color.colorPrimary)))
+                //.accentColor(sharedPref.getInt("theme_accent", ContextCompat.getColor(this, R.color.colorAccent)))
+                // .accentColor(Color.rgb(53, 58, 67))
+                // .backgroundColor(sharedPref.getInt("theme_background", ContextCompat.getColor(this, R.color.colorPrimary)))
```
Comment (davidnguyen2302) [inline tester/src/main/java/com/filestack/android/demo/MainActivity.java]: still need these commented out code?

**Review by davidnguyen2302 — APPROVED** [review]

File: tester/src/main/java/com/filestack/android/demo/MainActivity.java (hunk 2 above)
Comment (firoze) [reply tester/src/main/java/com/filestack/android/demo/MainActivity.java]: changed back

File: tester/src/main/java/com/filestack/android/demo/MainActivity.java (hunk 1 above)
Comment (firoze) [reply tester/src/main/java/com/filestack/android/demo/MainActivity.java]: nah, removed

**Review by j1george — APPROVED** [review]

**Pull request merged**